2024/ig-security.html

```diff
@@ -265,7 +265,7 @@ <h2 id="participation">
         <p>To be successful, this Interest Group is expected to include Security Researchers, Threat Modeling experts, Cryptographers, Cryptoanalysts, and active Editors for each deliverable. The Chairs and Editors are expected to contribute half of a working day per week. There is no minimum requirement for other Participants.</p>
 	<p>Participation in discussions via mailing lists and GitHub is free, as described in <a href="#communication">Communication</a>.</p>
 	<p>Participation in reviews, deliverable development, and meetings requires joining the group. The group welcomes and encourages all participants with proven specific expertise, even if they do not represent a W3C Member. In that case, they should join as <a href="https://www.w3.org/invited-experts/">Invited Experts</a>. Invited Experts in this group are not granted access to Member-only information.</p>
-	<p>The group also welcomes non-Members to contribute technical submissions for consideration upon their agreement to the terms of the <a href="https://www.w3.org/policies/patent-policy/">W3C Patent Policy</a>.</p>
+	<p>When a participant of this Interest Group contributes to a technical submission reviewing or marking comments on deliverables by other groups, agrees to the terms of the [W3C Patent Policy](https://www.w3.org/policies/patent-policy/) and [License Grants from Non-Participants](https://www.w3.org/policies/process/#contributor-license).</p>
```

Comment:
```suggestion
	<p>When a participant of this Interest Group contributes to a technical submission reviewing or marking comments on deliverables by other groups, they must agree to the terms of the [W3C Patent Policy](https://www.w3.org/policies/patent-policy/) and [License Grants from Non-Participants](https://www.w3.org/policies/process/#contributor-license).</p>
```

Reply:
I think this is the intent of the text proposed, but without this additional change, it seems to me that the grammar is a little off.